augly/image/functional.py

```diff
@@ -74,7 +74,7 @@ def apply_lambda(
         **func_kwargs,
     )
 
-    return imutils.ret_and_save_image(aug_image, output_path)
+    return imutils.ret_and_save_image(aug_image, output_path, src_mode)
```

Comment:
This looks like what we want to do, but you haven't defined `src_mode` yet. You can get the mode of the src image like this (be sure to add this line in each image function **before** `image` is augmented in any way but **after** `validate_and_load_image()` is called & **after** `get_func_kwargs()` is called (so `src_mode` isn't added to the metadata).
```python
src_mode = image.mode
```
(Source: https://stackoverflow.com/questions/33831572/get-image-mode-pil-python)